lib/Config.js

```diff
@@ -990,8 +990,11 @@ class Config extends EventEmitter {
             process.env[`${locationConstraint}_GCP_SERVICE_KEYFILE`];
         const serviceEmailFromEnv =
             process.env[`${locationConstraint}_GCP_SERVICE_EMAIL`];
-        const serviceKeyFromEnv =
+        let serviceKeyFromEnv =
             process.env[`${locationConstraint}_GCP_SERVICE_KEY`];
+        if (typeof serviceKeyFromEnv === 'string') {
+            serviceKeyFromEnv = serviceKeyFromEnv.replace(/\\n/g, '\n');
```

Comment:
I think this deserves a comment in the code telling why we do this.

Reply:
will add the comment